pkg/apis/edgefs.rook.io/v1/utils.go

```diff
@@ -40,7 +40,7 @@ func ByteCountBinary(b uint64) string {
 func GetModifiedRookImagePath(originRookImage, addon string) string {
 	imageParts := strings.Split(originRookImage, "/")
 	latestImagePartIndex := len(imageParts) - 1
-	modifiedImageName := "edgefs"
+	modifiedImageName := ""
```

Comment:
why?

Reply:
we have assigned this `modifiedImageName :=  "edgefs` but we are overwriting this value before using this variable. So, no need to assign value.

Reply:
In that case, this would be preferred instead of assigning to the empty string, although it's pretty much the same thing. While you're at it, you could move this declaration to line 51 right before it's actually used.
```suggestion
	var modifiedImageName string
```

Reply:
Done.